tests/unit/Tag/TagTitleTest.php

```diff
@@ -63,6 +63,8 @@ function () {
                 Tag::setTitle($value);
 
                 expect(Tag::getTitle())->equals("<title>{$value}</title>" . PHP_EOL);
+                
+                expect(Tag::getTitle(false))->equals("{$value}" . PHP_EOL);
```

Comment:
@Jurigag, @sergeyklay, @virgofx   Thank you for all help u provide to me. 
Thanks a lot!